ansible/roles/ssh/tasks/main.yml

```diff
@@ -11,3 +11,10 @@
               line="PasswordAuthentication no"
               state=present
   notify: Restart ssh
+
+- name: Set SSH logging level to VERBOSE
+  lineinfile: dest=/etc/ssh/sshd_config
+              regexp="^LogLevel"
+              line="LogLevel VERBOSE"
+              state=present
+  notify: Restart ssh
```

Comment:
This allows us to see which key was used to login to a given account. Log output is something like:
```
Jul 22 13:40:15 precise64 sshd[27718]: Connection from 192.168.33.14 port 37519
Jul 22 13:40:15 precise64 sshd[27718]: Found matching RSA key: 49:33:77:25:3e:05:ba:51:59:2a:c9:aa:f1:bd:24:29
Jul 22 13:40:15 precise64 sshd[27718]: Postponed publickey for dmiller from 192.168.33.14 port 37519 ssh2 [preauth]
Jul 22 13:40:15 precise64 sshd[27718]: Found matching RSA key: 49:33:77:25:3e:05:ba:51:59:2a:c9:aa:f1:bd:24:29
Jul 22 13:40:15 precise64 sshd[27718]: Accepted publickey for dmiller from 192.168.33.14 port 37519 ssh2
```
See also https://help.ubuntu.com/community/SSH/OpenSSH/Configuring#Log_More_Information